src/queryProcessor.js

```diff
@@ -143,6 +150,118 @@ export class QueryProcessor {
     };
   }
 
+  statsQuery(target, postData) {
+    if (target.seriesAggFn === 'none') {
+      return this.statsQueryUnmerged(target, postData);
+    }
+    const url = this.url + '/' + this.typeResources[target.type] + '/stats/query';
```

Comment:
If there is such complicated string, consider using [mplate_literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)
```javascript
const url = `${this.url}/${this.typeResources[target.type]}/stats/query`;
```